go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.10.0
 	go.opentelemetry.io/proto/otlp v0.19.0
 	go.uber.org/goleak v1.2.1
-	go.uber.org/zap v1.19.0
```

Comment:
the main consumer of zap is the etcd client... the version of the etcd client we use tests with `go.uber.org/zap v1.17.0`

is there a reason to move further away from the version our etcd client tests with? is this fixing issues or just bumping to latest versions?

Reply:
I don't know enough about the dependency to know if matching versions more closely is important.

Reply:
> @liggitt zap is already on `1.19.0` in the master now. :), so this is good to go.

are you referring to etcd master? looks like [zap is at 1.24.0 now on etcd client master](https://github.com/etcd-io/etcd/blob/main/client/v3/go.mod#L12), but you also need to look at whether etcd had to adjust usage of zap when transitioning to newer versions of zap